src/components/tables/requirements-table.tsx

```diff
@@ -3,19 +3,54 @@ import { ColumnDef } from '@tanstack/react-table'
 import { DataTable } from './data-table'
 import { RequirementApplication } from '@/constants/types'
 import { useGetRequirements } from '@/data/hooks/useRequirementsClient'
+import { Button } from '../ui/button'
+import { FileEdit, Trash2 } from 'lucide-react'
+import { useRouter } from 'next/navigation'
+import { PageRoutes } from '@/constants/page-routes'
+
+const ActionButtons = ({ row }: { row: any }) => {
+  const router = useRouter();
+  const requirement = row.original;
+
+  return (
+    <div className='flex items-center gap-2'>
+      <Button variant={"link"} onClick={() => router.push(`${PageRoutes.admin.REQUIREMENTS}/${requirement.id}`)}><FileEdit size={17} color='black' /></Button>
+      <Button variant={"ghost"}><Trash2 color='red' size={17} /> </Button>
+    </div>
+  )
+};
 
 export const columns: ColumnDef<RequirementApplication>[] = [
   {
     accessorKey: 'name',
     header: 'Name'
   },
   {
-    accessorKey: 'requirements',
-    header: 'Requirements'
+    accessorKey: 'incomeProfile',
+    header: 'Income Profile'
+  },
+  {
+    accessorKey: 'residenceType',
+    header: 'Residence Type'
+  },
+  {
+    header: 'Requirement Documents',
```

Comment:
Let's show these as a list ot tags.

Reply:
meaning?

Reply:
I mean a list of labels for better visual